docs/concepts/index.rst

```diff
@@ -214,10 +223,14 @@ Glossary
          See `Penalty method on Wikipedia <https://en.wikipedia.org/wiki/Penalty_method>`_
 
       Penalty model
-         An approach to solving constraint satisfaction problems (CSP) using an :term:`Ising` model 
-         or a :term:`QUBO` by mapping each individual constraint in the CSP to a ‘small’ Ising model 
+         An approach to solving constraint satisfaction problems (CSP) using an :term:`Ising` model
+         or a :term:`QUBO` by mapping each individual constraint in the CSP to a ‘small’ Ising model
          or QUBO.
 
+      Quadratic model
+         A collection of binary and/or integer variables with associated linear
```

Comment:
I think it's actually better to not restrict the variable types. Quadratic model just means that you have a set of variables with linear and quadratic relationships. Conceivably it could include continuous variables in the future, for instance.

Reply:
Changed to "A collection of variables with associated linear and "